lib/__tests__/standalone-formatter.test.mjs

```diff
@@ -57,3 +58,52 @@ it('standalone with input css and custom promised formatter', async () => {
 
 	expect(report).toContain('block-no-empty');
 });
+
+it('standalone with formatter specified in configuration', async () => {
+	const configWithFormatter = {
+		...configBlockNoEmpty,
+		formatter: 'string',
+	};
+
+	const { report } = await standalone({
+		code: 'a {}',
+		config: configWithFormatter,
+	});
+
+	const strippedOutput = stripAnsi(report);
+
+	expect(strippedOutput).toContain('1:3');
+	expect(strippedOutput).toContain('block-no-empty');
+});
+
+it('standalone with formatter argument overriding configuration formatter', async () => {
+	const configWithFormatter = {
+		...configBlockNoEmpty,
+		formatter: 'json',
+	};
+
+	const { report } = await standalone({
+		code: 'a {}',
+		config: configWithFormatter,
+		formatter: 'string',
+	});
+
+	const strippedOutput = stripAnsi(report);
+
+	expect(strippedOutput).toContain('1:3');
+	expect(strippedOutput).toContain('block-no-empty');
+});
+
+it('standalone with custom formatter function specified in configuration', async () => {
+	const configWithCustomFormatter = {
+		...configBlockNoEmpty,
+		formatter: customFormatter,
+	};
+
+	const { report } = await standalone({
+		code: 'a {}',
+		config: configWithCustomFormatter,
+	});
+
+	expect(report).toContain('errored: true');
+});
```

Comment:
[suggestion] It seems there is no case for `"formatter": "path/to/customformatter.js"`. Can you add it?